src/blocks/image.v1.yaml

```diff
@@ -21,14 +21,8 @@ definitions:
         allOf:
           - $ref: ../misc/asset.v1.yaml
           - properties:
-                alt:
-                    title: Alternative text
-                    type: string
-                uri:
-                    title: Image URI
-                    type: string
-                    format: uri
-                    pattern: ^https://
+                image:
```

Comment:
This is nuclear for the bot-lax-adaptor, could we prepare an intermediate version where:
- `alt` and `uri` are allowed but optional
- `image` is allowed but optional
- if it's easy: at least one of the two options (`alt`+`uri` or `image) is required
That would give the bot and lax complex time to migrate to the new format without breaking all of `prod--journal.elifesciences.org` images. It won't be immediate to implement DLCS integrations that are able to produce this `image` field.